### DISCUSSION TITLE
Too much padding between the metadata sidebar and the release card

### DISCUSSION THREAD

**Discussion by thislooksfun** [community/community#6435] [Other Feature Feedback, Questions, & Ideas] [answered]
There is a lot of space between the metadata (time, author, tag, etc) and the actual release notes, especially on wider displays. This makes the two columns feel very disconnected, and could easily lead to confusion if someone is trying to find some of that metadata and they don't think to looks to the side. Reducing the gap between the metadata and the card would go a long way towards making them feel connected.

**Comment by MylesBorins** [comment, marked as answer]
Good news, we just shipped an update to minimize that space. PTAL

**Reply by thislooksfun** [reply]
That looks much better, thank you!
